pkg/installer/installation/prerequisites.go

```diff
@@ -167,11 +168,24 @@ sudo yum install -y --disableexcludes=kubernetes \
 	kubectl-{{ .KUBERNETES_VERSION }}-0 \
 	kubernetes-cni-{{ .CNI_VERSION }}-0
 sudo systemctl enable --now docker
+sudo systemctl enable --now kubelet
 `
 
 	kubeadmCoreOSScript = `
 source /etc/kubeone/proxy-env
 
+# Short-Circuit the installation if it was already executed
+if type docker &>/dev/null && type kubelet &>/dev/null; then exit 0; fi
```

Comment:
nope, that shouldn't be here

Reply:
Why? Isn't `kubeone install` if run multiple times always going to reinstall kubelet? Don't we want to prevent that?

Reply:
oh well, yes, I didn't paid attention to what followed change lines (and was hidden).